changelog/6581.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed a bug where :func:`~sunpy.time.parse_time` would always disregard the remainder of a time string starting with the ifnal period if it was followed by only zeros, which could affect the parsing of the time string.
```

Comment:
```suggestion
Fixed a bug where :func:`~sunpy.time.parse_time` would always disregard the remainder of a time string starting with the finial period if it was followed by only zeros, which could affect the parsing of the time string.
```

Reply:
Heh, you misspelled your fix of my misspelling =)